operatorcert/entrypoints/upload_artifacts.py

```diff
@@ -143,8 +142,22 @@ def upload_logs(args: Any) -> Dict[str, Any]:
     Returns:
         Dict[str, Any]: Pyxis artifacts upload response
     """
-    LOGGER.info(f"Uploading logs: {args.log_file}")
-    return upload_artifact(args, args.log_file)
+    LOGGER.info(f"Uploading logs: {args.path}")
+    return upload_artifact(args, args.path)
+
+
+def upload_pipeline_logs(args: Any) -> Dict[str, Any]:
+    """
+    Upload tekton pipeline logs
+
+    Args:
+        args (Any): CLI arguments
+
+    Returns:
+        Dict[str, Any]: Pyxis artifacts upload response
+    """
+    LOGGER.info(f"Uploading pipeline logs: {args.path}")
+    return upload_artifact(args, args.path)
```

Comment:
This is almost identical to the `upload_logs` function. Couldn't we call `upload_artifacts` directly and remove these two functions? They don't seem to offer much value.

Reply:
Fixed